projects/plugins/crm/includes/ZeroBSCRM.AJAX.php

```diff
@@ -757,7 +757,8 @@ function ZeroBSCRM_get_quote_template() {
 				$quote_val = sanitize_text_field( wp_unslash( $_POST['quote_fields']['zbscq_value'] ) );
 			}
 			if ( isset( $_POST['quote_fields']['zbscq_date'] ) && ! empty( $_POST['quote_fields']['zbscq_date'] ) ) {
-				$quote_date = sanitize_text_field( wp_unslash( $_POST['quote_fields']['zbscq_date'] ) );
+				$sanitized_date = sanitize_text_field( wp_unslash( $_POST['quote_fields']['zbscq_date'] ) );
+				$quote_date     = wp_date( get_option( 'date_time' ), strtotime( $sanitized_date ) );
```

Comment:
This gets a bit dicey with timezones...I believe it will cause issues (though haven't tested).

Reply:
Not too sure what issues you mean here? I did make a mistake here though - it should have been `get_option( 'date_format' ). Fixing that did resolve issues I was now noticing when using some timezones, so perhaps that is what you meant? Commit: 07c425aee8ed834e67ea946545cf9811736f996a

Reply:
Following up - I have converted these to the jpcrm function now.